.travis.yml

```diff
@@ -17,6 +17,9 @@ jobs:
     - stage: platform-test
       node_js: "node"
       os: osx
+    - stage: platform-test
```

Comment:
We'll need this block for Linux too now that `node` points to 11

Reply:
Ah yeah totally forgot Travis is both linux and osx. I've been away too long.